index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Intuit Accounts - Sign Up</title>
+</head>
+
+  <header class="margin">
+    <!-- Sign Up-->
+    <div class="signup-text">
+      <p>Already have an Intuit account?</p>
+      <a href="https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Ftask%3DS#">
+    I forgot my User ID or Password</a>
+    </div>
+    <div class="signin-btn">  
+      <button class="submit" type="submit" onclick="window.location.href='https://accounts.intuit.com/index.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Ftask%3DS'">
+        <img class="padlock" src="assets/FAVPNG_padlock-icon-design-download_Utn0kFdE.png" alt="padlock">
+        
+          Sign In
+      </button>
+    </div>
+  </header>
+
+  <main>
+    <!-- Form -->
+    <section class="logotext">
+      <div class="logotext">
+        <img class="intuiticons1" src="assets/intuitlogo.png" alt="intuit">
+      </div>
+      <div class="logotext">
+      <img class="intuiticons2" src="assets/intuiticons.png" alt="intuiticon2">
+      </div>
+    </section>
+    
+    <form class="form">
+      <div class="textform">
+        <h1> Create an Intuit account</h1>
+        <p> One account for everything Intuit, including Mint.</p>
+        <a href="#">Learn more</a>
+      </div>
+      
+      <fieldset>
+        <label for="emailform">Email address<br>
+          <input class="emailform input-size" id="emailform" type="email" name="email">
+        </label><br>
+        <label for="phoneform">Phone (recommended)<br>
+          <input class="phoneform input-size" id="phoneform" type="number" name="phone">
```

Comment:
The `Phone Number` field should be of `type="tel"`, and not `type="number"`. Kindly change the type of input.

![22:27:56](https://user-images.githubusercontent.com/26856491/94488073-e2d3e280-01d9-11eb-823b-b61f59f80062.png)